### ISSUE TITLE
npm run develop fails after creating fresh new project with cli

### ISSUE THREAD

**Issue by alexstanbury** [gatsbyjs/gatsby#28613]
## Description

I followed the [quick start guide](https://www.gatsbyjs.com/docs/quick-start/) and can't get the `npm run develop` command to finish successfully unless I jump through a few hoops. I selected various options when I ran `npm init gatsby` which I presume should set up a default config for me. Instead, I got an error about no icon being set in the manifest plugin and then once I'd manually edited the config file I got another error from the file plugin saying the icon file didn't exist (understandably so)

To fix all this:

I have to edit the manifest plugin settings manually by browsing to your docs and copy pasting a default config in.
I have to manually add an icon property to this config block.
I have to create an 'images' folder inside 'src'.
I have to add an icon.ong file into that folder.

### Steps to reproduce

1. `npm init gatsby`
2. don't select a CMS
3. don't select styling option, 
4. select all plugin options except the two manifest options
5. wait for install to finish and cd into dir
6. run `npm run develop`
7. ???????
8. observe build errors

### Expected result

I should be able to successfully run `npm run develop` and not get any errors.

### Actual result

I got multiple errors.

### Environment

  System:
    OS: macOS 10.15.7
    CPU: (4) x64 Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.16.3 - ~/.nvm/versions/node/v12.16.3/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.4 - ~/.nvm/versions/node/v12.16.3/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 87.0.4280.88
    Firefox: 75.0
    Safari: 14.0
  npmPackages:
    gatsby: ^2.26.1 => 2.28.2 
    gatsby-plugin-google-analytics: ^2.7.0 => 2.7.0 
    gatsby-plugin-image: ^0.3.1 => 0.3.1 
    gatsby-plugin-manifest: ^2.8.0 => 2.8.0 
    gatsby-plugin-offline: ^3.6.0 => 3.6.0 
    gatsby-plugin-react-helmet: ^3.6.0 => 3.6.0 
    gatsby-plugin-sharp: ^2.10.1 => 2.10.1 
    gatsby-plugin-sitemap: ^2.8.0 => 2.8.0 
    gatsby-source-filesystem: ^2.7.0 => 2.7.0 
    gatsby-transformer-sharp: ^2.8.0 => 2.8.0 
  npmGlobalPackages:
    gatsby-cli: 2.15.1

**Comment by LekoArts** [comment]
Thanks for the issue! We're aware of it and will see to resolve this soon!

**Comment by thatjimmy** [comment]
I just used `npm init gatsby` as is recommended from the [quick start page](https://gatsbyjs.com/docs/quick-start/). I've used gatsby on a few projects before but this is my first time using `npm init gatsby`. I choose styled components, responsive images, and react helmet as options. Anyway, no plugins were installed, and gatsby-config.js is empty. Git wasn't initialized either. Sounds like same problem as op? Thanks!
